overview.md

```diff
@@ -33,80 +31,91 @@ search across any set of indexed STAC [`Items`](stac-spec/item-spec/README.md).
+| **`/collections`**                                  | [OAFeat](ogcapi-features)  | data              | JSON                                              | Object with a list of Collections contained in the catalog and links                                                                |
+| **`/conformance`**                                  | [OAFeat](ogcapi-features)  | conformance       | JSON                                              | Info about standards to which the API conforms                                                                                      |
+| `/api`                                              | [OAFeat](ogcapi-features)  | service-desc      | OpenAPI 3.0 JSON                                  | The OpenAPI definition of the endpoints in this service                                                                             |
+| **`/collections/{collectionId}`**                   | [OAFeat](ogcapi-features)  | collection        | Collection                                        | Returns single Collection JSON                                                                                                      |
+| **`/collections/{collectionId}/items`**             | [OAFeat](ogcapi-features)  | items             | ItemCollection                                    | GeoJSON FeatureCollection-conformant entity of Items in collection                                                                  |
+| **`/collections/{collectionId}/items/{featureId}`** | [OAFeat](ogcapi-features)  | Item              | Returns single Item (GeoJSON Feature)             |                                                                                                                                     |
 
 ## Conformance Classes
 
 STAC API is evolving to utilize OAFeat's 
 '[Conformance](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_declaration_of_conformance_classes)' JSON structure. For 
 STAC API 1.0.0-beta.1 we declare new STAC Conformance classes, and specify which OAFeat ones to use. These are detailed in the relevant
-sections below.The core STAC conformance classes communicate the conformance JSON only in the root (`/`) document, while OGC API 
-requires they also live at the `/conformance` endpoint. 
+sections below. The core STAC conformance classes communicate the conformance JSON only in the root (`/`) document, while OGC API 
+requires they also live at the `/conformance` endpoint. STAC's conformance structure is detailed in the [core](core/).
 
 **NOTE:** *By 1.0.0 we aim to have requirements classes specified in detail, as testable assertions, 
 like OGC does, but for now the core reference is just this spec document and the OpenAPI yaml. We also desire to have the
```

Comment:
Isn't Matt's PR #86 already publishing the OpenAPIs to the conformance class URIs? At least that's what I hope it does. It doesn't make sense to have them at two places.

Reply:
Oh, by 'conformance structure' I just meant the full description of STAC's conformsTo JSON construct. I'll reword here.

Reply:
I think this is a misunderstanding, I was referring to:

>  We also desire to have the URI's for conformance to actually resolve to machine-readable information clients can use.

I thought that is what Matt is working on?

Reply:
Ah, gotcha - I was looking at the green highlight. 

>> We also desire to have the URI's for conformance to actually resolve to machine-readable information clients can use.

> I thought that is what Matt is working on?

I thought he was just publishing html versions online. Not sure if he was trying to make them so they'd be the same location and resolve. I was considering that out of scope for this release, but I'm not opposed to it.

Reply:
I did not fully understood your conclusion yesterday (non-native speaker issues ;-) ), but I think we should publish the OpenAPIs and HTML versions... And for the HTML versions you usually need the OpenAPIs. But anyway, it doesn't matter here to what it resolves, it gives you the information anyway. cc @matthewhanson

Reply:
Ah. My conclusion was we should just have the OpenAPI's in github, and publish the HTML directly off of those. But I don't feel strongly.

Reply:
I'm publishing the yaml files to gh-pages in this repo. 
Then we can use ReDoc to point to any of them to create an html version, like in here: https://github.com/radiantearth/stac-site/blob/master/STAC-api.html